scout/server/blueprints/panels/templates/panels/panels.html

```diff
@@ -96,20 +96,60 @@
         <!--create table for each panel-->
         <tbody>
           <tr>
-            <td><a href="{{ url_for('panels.panel', panel_id=panel._id) }}">{{ panel.display_name }}</a></td>
+            <td>
+              <a href="{{ url_for('panels.panel', panel_id=panel._id) }}">{{ panel.display_name }}</a>
+              {% if panel.hidden %}
+                <span class="badge badge-danger">Removed</span>
+              {% endif %}
+            </td>
             <td>{{ panel.version }} ({{ panel.date.date()}})</td>
             <td><span class="badge badge-pill badge-secondary">{{ panel.genes|length}}</span></td>
             <td><button id="{{panel._id}}" type="button" data-toggle="collapse" href="#paneldiv_{{panel._id}}" aria-expanded="false" aria-controls="paneldiv_{{panel._id}}" class="btn btn-primary btn-xs"><span class="fa fa-search-plus"></span></button><br>{{ history_view(panel._id, panel.panel_name) }}</td>
-            <td><button id="{{panel._id}}" type="button" data-toggle="collapse" href="#modifydiv_{{panel._id}}" aria-expanded="false" aria-controls="modifydiv_{{panel._id}}" class="btn btn-warning btn-xs"><span class="fa fa-pencil-square-o"></span></button><br>{{ history_view(panel._id, panel.panel_name) }}</td>
+            <td><button id="{{panel._id}}" type="buttonjj" data-toggle="collapse" href="#modifydiv_{{panel._id}}" aria-expanded="false" aria-controls="modifydiv_{{panel._id}}" class="btn btn-warning btn-xs"><span class="fa fa-pencil-square-o"></span></button><br>{{ history_view(panel._id, panel.panel_name) }}</td>
+            <td>
+              {% if panel.hidden %}
+                <form action="{{ url_for('panels.panel_restore', panel_id=panel._id) }}" method="POST">
+                  <button id="{{panel._id}}" type="submit" title="Restore panel" class="btn btn-success btn-xs"><span class="fa fa-undo"></span></button>
+                </form>
+              {% else %}
+                <button id="{{panel._id}}" type="button" data-toggle="modal" data-target="#removeGenePanelModal" class="btn btn-danger btn-xs"><span class="fa fa-trash"></span></button>
```

Comment:
The functionality works nicely but I am not convinced that any user should be allowed to remove gene panels..